projects/pgai/pgai/cli.py

```diff
@@ -58,11 +59,35 @@ def get_bool_env(name: str | None) -> bool:
 tracer.enabled = get_bool_env("DD_TRACE_ENABLED")
 
 
-def get_pgai_version(cur: psycopg.Cursor) -> str | None:
+class Version:
+    def __init__(self, ext_version: str | None, app_version: str | None):
+        self.ext_version = ext_version
+        self.app_version = app_version
+
+    def __str__(self):
+        return f"ext_version: {self.ext_version}, app_version: {self.app_version}"
```

Comment:
You could use a dataclass for this:

```suggestion
@dataclass
class Version:
    ext_version: str | None = None
    app_version: str | None = None

    def __str__(self):
        return f"ext_version: {self.ext_version}, app_version: {self.app_version}"
```